Source/Extensions/Blazorise.Animate/Blazorise.Animate.csproj

```diff
@@ -3,20 +3,10 @@
   <Import Project="..\..\..\Build\Blazorise.props" />
 
   <PropertyGroup>
-    <PackageId>Blazorise.Animate</PackageId>
```

Comment:
What happens with PackageId? Is it going to be automatically generated based on project name?

Reply:
yep it defaults to the project name